src/main/java/com/bobocode/hoverla/bring/context/ApplicationContextImpl.java

```diff
@@ -1,60 +1,152 @@
 package com.bobocode.hoverla.bring.context;
 
+import com.bobocode.hoverla.bring.exception.NoSuchBeanException;
+import com.bobocode.hoverla.bring.exception.NoUniqueBeanException;
+import com.google.common.collect.HashBasedTable;
+import com.google.common.collect.Table;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.Objects;
+
 
 /**
- * Example of ApplicationContext implementation, can be removed or reimplemented when actual development starts
+ * Implementation of ApplicationContext
  */
+@Slf4j
 public class ApplicationContextImpl implements ApplicationContext {
+    private Table<String, Class<?>, BeanDefinition> context = HashBasedTable.create();
```

Comment:
I would rename this field to `beanDefinitionTable`

Reply:
@NazarSmith228 Done

Reply:
@NazarSmith228 Done